libs/sdk-ui-kit/src/List/InsightListItemDate.tsx

```diff
@@ -71,15 +76,19 @@ export function getDateTimeConfig(
 ): IInsightListItemDateConfig {
     const { dateTimezone = META_DATA_TIMEZONE, now = new Date() } = options;
 
-    const dateInLocalTimezone = toDate(date, { timeZone: dateTimezone });
-    const yesterday = subDays(now, 1);
+    const dateWithTimezone = moment.tz(date, dateTimezone);
+    const dateWithTimezoneInLocal = dateWithTimezone.clone().local();
+
+    const NOW = moment(now);
+    const TODAY = moment(now);
+    const YESTERDAY = moment(now).subtract(1, "days");
 
-    const isToday = isSameDay(dateInLocalTimezone, now);
-    const isYesterday = isSameDay(dateInLocalTimezone, yesterday);
-    const isCurrentYear = isSameYear(dateInLocalTimezone, now);
+    const isToday = sameFormatted(dateWithTimezoneInLocal, TODAY, FORMAT_DATE);
```

Comment:
Can we please create a follow up for this so that it avoids the `sameFormatted` like the previous solution? `moment` has the [`isSame`](https://momentjs.com/docs/#/query/is-same/) function whihc should do the same thing more idiomatically.

Reply:
yes, created, ONE-5001